### PR TITLE
Added support for NTAG215

### PR DESCRIPTION
**Pull request by gcammisa**
This pull request adds partial support for NTAG215 emulation.

The two important things that are still not implemented are:
1) Bruteforce "protection" for the PWD_AUTH command.
2) The emulated card does not enforce protection set by static and dynamic lockbytes.

### CODE REVIEW

**Comment by ca1e** [comment]
great!could not wait for it.But i just not sure if it could works on revE rebooted.

**Comment by shrmnk** [comment]
Hope this gets merged!

File: Firmware/Chameleon-Mini/Application/NTAG215.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#ifndef NTAG215_H_
+#define NTAG215_H_
+
+#include "Application.h"
+#include "ISO14443-3A.h"
+
+#define NTAG215_UID_SIZE ISO14443A_UID_SIZE_DOUBLE //7 bytes UID
+#define NTAG215_PAGE_SIZE 4 //bytes per page
+#define NTAG215_PAGES 135 //135 pages total, from 0 to 134
+#define NTAG215_MEM_SIZE ( NTAG215_PAGE_SIZE * NTAG215_PAGE_SIZE )
```
Comment (HexRabbit) [inline Firmware/Chameleon-Mini/Application/NTAG215.h:17]: The `NTAG215_MEM_SIZE` here should be `NTAG215_PAGE_SIZE * NTAG215_PAGES`.

**Comment by gcammisa** [comment]
The EM4233 page that you referred me to is mostly documentation regarding 
info bits and lock status bytes.

The "equivalent" locking mechanism of NTAG215 (static/dynamic lockbytes) is not implemented
in this version of the NTAG215 application, so honestly I wouldn't really know what to write
in such documentation.
The dump that the application expects is just a standard NTAG215 dump.

File: Firmware/Chameleon-Mini/Application/NTAG215.h (hunk 1 above)
Comment (gcammisa) [reply Firmware/Chameleon-Mini/Application/NTAG215.h:17]: This is for sure a mistake on my part, I probably made it while "cleaning" the code and didn't notice it.
Thank you for pointing it out, I fixed it in my repo.
I'm not completely familiar with how pull request reviews work so, if there's anything else I should do to "fix" the pull request, please let me know

**Pull request merged**